features/cellular/framework/API/CellularPower.h

```diff
@@ -92,7 +92,7 @@ class CellularPower
      *
      *  @return zero on success
      */
-    virtual nsapi_error_t set_power_level(int func_level, int do_reset = 1) = 0;
+    virtual nsapi_error_t set_power_level(int func_level, int do_reset = 0) = 0;
```

Comment:
what is this fixing - changing the default value?

Reply:
So that it's not an API change we used default value but it was wrong in this pr: https://github.com/ARMmbed/mbed-os/pull/7146
Here it's corrected and now it does not change the API behaviour.